packages/code-studio/src/styleguide/CheckboxGroups.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { Checkbox, CheckboxGroup, Flex, Text } from '@deephaven/components';
+import SampleSection from './SampleSection';
+
+export function CheckboxGroups(): JSX.Element {
+  return (
+    <SampleSection name="checkbox-groups">
+      <h2 className="ui-title">Checkbox Groups</h2>
+      <Flex gap="size-100" gridColumn="span 3" height="100%">
+        <Flex direction="column">
+          <Text>Single Child</Text>
+          <CheckboxGroup aria-label="Single Child">
+            <Checkbox checked={false}>Aaa</Checkbox>
+          </CheckboxGroup>
+        </Flex>
+
+        <Flex direction="column">
+          <Text>Multiple Children</Text>
+          <CheckboxGroup aria-label="Multiple Children">
+            <Checkbox checked={false}>Aaa</Checkbox>
+            <Checkbox checked={false}>Bbb</Checkbox>
+            <Checkbox checked={false}>Ccc</Checkbox>
+          </CheckboxGroup>
+        </Flex>
+
+        <Flex direction="column">
+          <Text>Mixed Children Types</Text>
+          <CheckboxGroup aria-label="Mixed Children Types">
+            {/* eslint-disable react/jsx-curly-brace-presence */}
+            {'String 1'}
+            {'String 2'}
+            {444}
+            {999}
+            {true}
```

Comment:
Looking at the styleguide, I think there's a bug with handling of `true` / `false`

<img width="374" alt="image" src="https://github.com/user-attachments/assets/7fded2a7-8602-4f7b-8463-9167cacfa17e">

There's probably a stringify that needs to happen somewhere

Reply:
When I wrap my children in wrappedChildren, I pass in `String(child) `into the checkbox, which seems to work for the other primitives. Is there any other way I can stringify?

Reply:
I think the issue here is that `React.Children.map` converts `true` / `false` to null.

Reply:
You might try `ensureArray` from @deephaven/utils instead

Reply:
Actually, you may still need React.Children.map to deal with `key` props, but you'll need to convert booleans to strings before passing into it.

Reply:
I think something like this will work:
```
const wrappedChildren = useMemo(
  () =>
    ensureArray(children).map(child =>
      isElementOfType(child, Checkbox) ? (
        child
      ) : (
        <Checkbox key={String(child)} value={String(child)}>
          {String(child)}
        </Checkbox>
      )
    ),
  [children]
);
```